src/utils/simplified-account/validation/worldpay/validations.schema.js

```diff
@@ -0,0 +1,48 @@
+const { body } = require('express-validator')
+
+const ONE_OFF_CUSTOMER_INITIATED_SCHEMA = {
```

Comment:
I'm not sure why the validations have to be nested inside the properties here but for THREE_DS_FLEX_VALIDATION they are not.

Reply:
So the ONE_OFF_CUSTOMER_INITIATED_SCHEMA is not a validation I wrote and [each property seems to be used individually for example](https://github.com/alphagov/pay-selfservice/blob/master/src/controllers/simplified-account/settings/switch-psp/switch-to-worldpay/add-worldpay-credentials.controller.js#L28-L30). Whereas with the THREE_DS_FLEX_VALIDATION I'm doing the validation [all in one shot](https://github.com/alphagov/pay-selfservice/blob/master/src/controllers/simplified-account/settings/switch-psp/switch-to-worldpay/add-flex-credentials.controller.js#L23).